en/chromebook_setup/instructions.md

```diff
@@ -61,6 +61,77 @@ This bottom area is your terminal. You can use the terminal to send instructions
 to the remote Cloud 9 computer. You can resize that window to make it a bit
 bigger.
 
+#### Glitch.com Cloud IDE
+
+1. Go to [Glitch.com](https://glitch.com/)
+2. Sign up for an account (https://glitch.com/signin) or use your Github account if you have one (see Github instructions below).
+3. Click _New Project_ and choose _Hello Webpage_
```

Comment:
This seems to be labeled "hello-webpage" not "Hello Webpage".

Also, on a fresh account, one doesn't have to click on "New Project" (at the upper right) to find that, it's also available at the front-and-center "Let's get started!" section, which might be easier to spot:
![glitch.com screenshot with "hello-website" highlighted by a red circle](https://user-images.githubusercontent.com/97746/112490545-be6b5400-8d7f-11eb-9822-e9df25b6fb6e.png)

```suggestion
3. Click _New Project_ and choose _hello-webpage_
```
or simply
```suggestion
3. Click on _hello-webpage_
```